cgeglio/rancid-tomatillos
# Unit testing

### What does this PR do?
- implement loading screen to display while movies are fetched
- create README.md
- add tests for MoviesContainer

### Where should the reviewer start?
- while viewing all movies, try refreshing the page to see loading screen displayed for a moment
- open README.md
- run tests for MoviesContainer

### How should this be manually tested?
- observe loading screen as movie data is fetched
- read README.md
- confirm most tests pass and appropriate functionality is tested in MovieContainer

### Potential changes?
- need to refactor a few test in MovieContainer
